docs/Coding-Guidelines/PullRequest.md

```diff
@@ -0,0 +1,10 @@
+# Pull Request
+When your code is in tip-top shape it is time to make a Pull Request (PR) into main.  A PR is a
+formal request to merge your code into main.
+
+**Must** Run the build pipeline by commenting `/azp run`.
```

Comment:
Also add a section about this.
'/azp run' will run all the pipelines associated with the repo. 
Doing '/azp run <Name of the pipeline>' will run specific pipelines.
This will come handy because with the develop branch, we will have multiple pipelines validating. When I was testing the develop branch, doing '/azp run' try to run both the internal and public pipelines.

Reply:
I'm confused.  The first line says `/azp run` runs all pipelines with the repo.  The next line says `/azp run` runs a specific pipeline.

Reply:
What do you mean by "All the pipelines?  @kythant